tests/test_nodes.py

```diff
@@ -417,6 +419,27 @@ def test_data_read(self):
 
         self.assertEqual(15, node.get_data_input("temperature"))
 
+    def test_data_overrides(self):
```

Comment:
This only test the first branch of the conditional statement. You need another test for the case of `not content` and another one for the `else` case, which is an error.

Reply:
Will add one. Just wonder how to verify the error message in the test function?